x/staking/types/keys.go

```diff
@@ -68,7 +68,8 @@ var (
 	BlockConsPubKeyRotationHistoryKey           = collections.NewPrefix(102) // prefix for consPubkey rotation history by height
 	ValidatorConsensusKeyRotationRecordQueueKey = collections.NewPrefix(103) // this key is used to set the unbonding period time on each rotation
 	ValidatorConsensusKeyRotationRecordIndexKey = collections.NewPrefix(104) // this key is used to restrict the validator next rotation within waiting (unbonding) period
-	RotatedConsKeyMapIndex                      = collections.NewPrefix(105) // prefix for rotated cons address to new cons address
+	NewToOldConsKeyMap                          = collections.NewPrefix(105) // prefix for rotated cons address to new cons address
+	OldToNewConsKeyMap                          = collections.NewPrefix(106) // prefix for rotated cons address to new cons address
```

Comment:
do we need two maps here?

Reply:
`NewToOldConsKeyMap` this is being used to handle malicious behaviour of the validator.
ref: https://github.com/cosmos/cosmos-sdk/pull/15165/files#diff-f772c3375ecfb8bef2d8654b49a8fd1a7d9bbba7a5d1c46f6cf471b2170075acR258

`OldToNewConsKeyMap` is handle the missed blocks map.